src/styl/now-value.styl

```diff
@@ -0,0 +1,22 @@
+@import '_variables'
+
+.dollert-now-value-container
+  padding-bottom: 15px
+  text-align: center
+  line-height: 20px
+  background $color-white
+  border-bottom 1px solid $color-grey
+  .dollar-now-value-label
+    font-size: 12px
+  .dollar-now-value-field
```

Comment:
Other question is about `dollert` and `dollar`. I used to prefix the css classes for this project with `dollert`. So, in this stylesheet, it's not clear for me if you wanted to prefix css classes with `dollert` or if you just wanted to do it for the first declared class.